addons/project/models/res_partner.py

```diff
@@ -8,9 +8,8 @@
 class ResPartner(models.Model):
     """ Inherits partner and adds Tasks information in the partner form """
     _inherit = 'res.partner'
-    _check_company_auto = True
 
-    project_ids = fields.One2many('project.project', 'partner_id', string='Projects', check_company=True)
+    project_ids = fields.One2many('project.project', 'partner_id', string='Projects')
```

Comment:
Can we add a python contraint to avoid having a partner in company A and set in project in company B for instance? 🤔 

Because the `check_company` we added was really to avoid that. 🙂 

Can you check? 🙂 

Thanks in advance! 🙂

Reply:
done Thanks for the feedback :)